metabase/metabase
# [Epic #35961] Milestone 4 - Add Legalese Step

Milestone 4 of Epic #35961

### Description

Adds the Legalese step for OSS and EE instances with no token

### How to verify

_In an OSS instances and EE instances without a valid token_
* Go to a dashboard/question and access the Embed modal (click the share icon and click the "Embed" option in the menu)
* Click into `Static Embed`
* You should see a screen containing the licensing terms for static embedding. Clicking `Agree and continue` should take you to the static embedding configuration screen.
* Refresh the page and try the same flow. You should immediately see the options for static embedding, as the user has already accepted the terms

_In an EE instance with a valid token_
* Go to a dashboard/question and access the Embed modal (click the share icon and click the "Embed" option in the menu)
* Click into `Static Embed`
* You should immediately see the options for static embedding. For EE instances, there is no legalese step


### Checklist
